<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>527</width>
                <height>402</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Data Producer</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <widget class="QGroupBox" name="serverForm">
                        <property name="minimumSize">
                            <size>
                                <width>281</width>
                                <height>111</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="title">
                            <string>Server IP</string>
                        </property>
                        <widget class="QPushButton" name="connectBtn">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>70</y>
                                    <width>121</width>
                                    <height>25</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Connect</string>
                            </property>
                        </widget>
                        <widget class="QPushButton" name="disconnectBtn">
                            <property name="geometry">
                                <rect>
                                    <x>150</x>
                                    <y>70</y>
                                    <width>121</width>
                                    <height>25</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Disconnect</string>
                            </property>
                        </widget>
                        <widget class="QLineEdit" name="server">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>30</y>
                                    <width>261</width>
                                    <height>31</height>
                                </rect>
                            </property>
                        </widget>
                    </widget>
                </item>
                <item row="0" column="1" rowspan="2">
                    <widget class="QGroupBox" name="log">
                        <property name="minimumSize">
                            <size>
                                <width>221</width>
                                <height>381</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="title">
                            <string>Logs</string>
                        </property>
                        <widget class="QTextBrowser" name="logScreen">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>30</y>
                                    <width>201</width>
                                    <height>341</height>
                                </rect>
                            </property>
                        </widget>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QGroupBox" name="dataController">
                        <property name="minimumSize">
                            <size>
                                <width>281</width>
                                <height>251</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="title">
                            <string>Data Controller</string>
                        </property>
                        <widget class="QLCDNumber" name="minLcd">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>40</y>
                                    <width>64</width>
                                    <height>23</height>
                                </rect>
                            </property>
                            <property name="styleSheet">
                                <string notr="true"/>
                            </property>
                            <property name="frameShape">
                                <enum>QFrame::Box</enum>
                            </property>
                            <property name="frameShadow">
                                <enum>QFrame::Plain</enum>
                            </property>
                            <property name="lineWidth">
                                <number>1</number>
                            </property>
                            <property name="smallDecimalPoint">
                                <bool>false</bool>
                            </property>
                            <property name="segmentStyle">
                                <enum>QLCDNumber::Flat</enum>
                            </property>
                        </widget>
                        <widget class="QLabel" name="minLabel">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>20</y>
                                    <width>67</width>
                                    <height>17</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Min</string>
                            </property>
                        </widget>
                        <widget class="QLCDNumber" name="maxLcd">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>100</y>
                                    <width>64</width>
                                    <height>23</height>
                                </rect>
                            </property>
                            <property name="frameShape">
                                <enum>QFrame::Box</enum>
                            </property>
                            <property name="frameShadow">
                                <enum>QFrame::Plain</enum>
                            </property>
                            <property name="segmentStyle">
                                <enum>QLCDNumber::Flat</enum>
                            </property>
                            <property name="intValue" stdset="0">
                                <number>1</number>
                            </property>
                        </widget>
                        <widget class="QLabel" name="maxLabel">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>80</y>
                                    <width>67</width>
                                    <height>17</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Max</string>
                            </property>
                        </widget>
                        <widget class="QLCDNumber" name="timingLcd">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>160</y>
                                    <width>64</width>
                                    <height>23</height>
                                </rect>
                            </property>
                            <property name="frameShape">
                                <enum>QFrame::Box</enum>
                            </property>
                            <property name="frameShadow">
                                <enum>QFrame::Plain</enum>
                            </property>
                            <property name="segmentStyle">
                                <enum>QLCDNumber::Flat</enum>
                            </property>
                            <property name="intValue" stdset="0">
                                <number>1</number>
                            </property>
                        </widget>
                        <widget class="QLabel" name="timingLabel">
                            <property name="geometry">
                                <rect>
                                    <x>210</x>
                                    <y>140</y>
                                    <width>71</width>
                                    <height>17</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Timing (s)</string>
                            </property>
                        </widget>
                        <widget class="QSlider" name="timingSlider">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>160</y>
                                    <width>191</width>
                                    <height>16</height>
                                </rect>
                            </property>
                            <property name="minimum">
                                <number>0</number>
                            </property>
                            <property name="maximum">
                                <number>60</number>
                            </property>
                            <property name="value">
                                <number>1</number>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                        <widget class="QSlider" name="maxSlider">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>100</y>
                                    <width>191</width>
                                    <height>16</height>
                                </rect>
                            </property>
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="value">
                                <number>1</number>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                        <widget class="QSlider" name="minSlider">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>40</y>
                                    <width>191</width>
                                    <height>16</height>
                                </rect>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                        <widget class="QPushButton" name="startBtn">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>200</y>
                                    <width>121</width>
                                    <height>25</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Start</string>
                            </property>
                        </widget>
                        <widget class="QPushButton" name="stopBtn">
                            <property name="geometry">
                                <rect>
                                    <x>150</x>
                                    <y>200</y>
                                    <width>121</width>
                                    <height>25</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Stop</string>
                            </property>
                        </widget>
                    </widget>
                </item>
            </layout>
            <zorder>dataController</zorder>
            <zorder>log</zorder>
            <zorder>serverForm</zorder>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections>
        <connection>
            <sender>minSlider</sender>
            <signal>valueChanged(int)</signal>
            <receiver>minLcd</receiver>
            <slot>display(int)</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>99</x>
                    <y>117</y>
                </hint>
                <hint type="destinationlabel">
                    <x>221</x>
                    <y>121</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>timingSlider</sender>
            <signal>valueChanged(int)</signal>
            <receiver>timingLcd</receiver>
            <slot>display(int)</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>160</x>
                    <y>237</y>
                </hint>
                <hint type="destinationlabel">
                    <x>221</x>
                    <y>241</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>maxSlider</sender>
            <signal>valueChanged(int)</signal>
            <receiver>maxLcd</receiver>
            <slot>display(int)</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>100</x>
                    <y>177</y>
                </hint>
                <hint type="destinationlabel">
                    <x>221</x>
                    <y>181</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
